From 3bbf579dde878842f96b4d3ac5fdf913d314516e Mon Sep 17 00:00:00 2001
From: Li Wang <li.wang@windriver.com>
Date: Fri, 24 Apr 2015 15:23:27 +0800
Subject: [PATCH 5/9] Revert "arch/powerpc/kernel: Address IBM PPC476 erratum
 47"

git://git.yoctoproject.org/linux-yocto-3.4
commit ac56a3f4f0532da36e5e5b90991fde69ccd87658

This reverts commit d49a60bb3e7545b772f285d4b25039e075e2cb3c.

Signed-off-by: Li Wang <li.wang@windriver.com>
---
 arch/powerpc/kernel/head_44x.S |    6 ++----
 1 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/arch/powerpc/kernel/head_44x.S b/arch/powerpc/kernel/head_44x.S
index 9f16b78..ac36884 100644
--- a/arch/powerpc/kernel/head_44x.S
+++ b/arch/powerpc/kernel/head_44x.S
@@ -1247,12 +1247,10 @@ clear_utlb_entry:
 
 	/* We configure icbi to invalidate 128 bytes at a time since the
 	 * current 32-bit kernel code isn't too happy with icache != dcache
-	 * block size. We also disable the BTAC as this can cause errors
-	 * in some circumstances (see IBM Erratum 47).
-	*/
+	 * block size
+	 */
 	mfspr	r3,SPRN_CCR0
 	oris	r3,r3,0x0020
-	ori 	r3,r3,0x0040
 	mtspr	SPRN_CCR0,r3
 	isync
 
-- 
1.7.5.4

